### PR TITLE
removed the min unity release rule

### PR DESCRIPTION
**Pull request by StephenHodgson**
# XRTK - Mixed Reality Toolkit Pull Request

## Overview

Removed the minimum unity release version for all XRTK projects. Any version of Unity 2019.4.X should be compatible.

### CODE REVIEW

File: XRTK-Core/Packages/manifest.json [hunk 1]
```diff
@@ -15,8 +15,6 @@
     "com.unity.ide.vscode": "1.2.3",
     "com.unity.mobile.android-logcat": "1.2.1",
     "com.unity.test-framework": "1.1.24",
-    "com.unity.textmeshpro": "2.1.4",
-    "com.unity.ugui": "1.0.0",
```
Comment (SimonDarksideJ) [inline XRTK-Core/Packages/manifest.json:19]: Does this not cause issue, removing the Unity UI system?  Granted Core doesn't use it

File: XRTK-Core/Packages/packages-lock.json [hunk 2]
```diff
@@ -62,7 +62,7 @@
     },
     "com.unity.textmeshpro": {
```
Comment (SimonDarksideJ) [inline XRTK-Core/Packages/packages-lock.json:63]: Are these still needed in the .lock file if they have been removed from the manifest?

Comment (StephenHodgson) [reply XRTK-Core/Packages/packages-lock.json:63]: Lock file is automatically updated by unity and keeps a lost of the direct dependencies in the project.

File: XRTK-Core/Packages/manifest.json (hunk 1 above)
Comment (StephenHodgson) [reply XRTK-Core/Packages/manifest.json:19]: No bc the dependency is defined in the project.json

Comment (StephenHodgson) [reply XRTK-Core/Packages/manifest.json:19]: Just means one less place to have to update the information

**Review by SimonDarksideJ — APPROVED** [review]

**Pull request merged**